.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "es6": true,
+    "webextensions": true
+  },
+
+  "extends": [
+    "eslint:recommended"
```

Comment:
Eventually, it may be nice to extend the "plugin:mozilla/recommended" rules too and use the ["mozilla" ESLint plugin config](http://npm.im/eslint-plugin-mozilla).